packages/maskbook/src/database/global.d.ts

```diff
@@ -1,18 +1,6 @@
 declare module 'idb/with-async-ittr-cjs' {
     export * from 'idb'
 }
-declare module 'metamask-extension-provider' {
```

Comment:
It does not make sense `module 'metamask-extension-provider'` locates in `database/`, I move this part to `EthereumServices/types.d.ts`